app/models/person.rb

```diff
@@ -0,0 +1,11 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def delete_associated_tasks
+    Task.where(person_id: self.id).destroy_all
```

Comment:
with the `has_many` association these `Task.where(person_id: self.id)` can be rewritten as just `tasks`.